sdk/textanalytics/ai-text-analytics/CHANGELOG.md

```diff
@@ -1,5 +1,20 @@
 # Release History
 
+## 1.0.0-preview.2 (2020-02-11)
+
+- Overhauled Text Analytics samples packages. #7052
```

Comment:
Better mention what's improved in samples regarding user experience.  If it's just the infrastructure work I don't think it's worth to list here.